### PR TITLE
Small auth and topics pages bugfix

### PR DESCRIPTION
**Pull request by ayomanetsan**
1. According to PO, upon registration user should be able to select language levels with description;
2. Small font changes on topic page;
3. Show/hide icon is now aligned with arrow icons.

### CODE REVIEW

**Review by ishootdice — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

File: frontend/src/app/modules/auth/sign-up/sign-up.component.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ export class SignUpComponent extends BaseComponent implements OnInit {
             age: this.dateOfBirth.value,
             sex: this.sex.value,
             language: this.language.value,
-            languageLevel: this.languageLevel.value,
+            languageLevel: (this.languageLevel.value as string).substring(0, 2),
```
Comment (tatianahutii) [inline frontend/src/app/modules/auth/sign-up/sign-up.component.ts:125]: Level should have an enum type, not string

File: frontend/src/app/shared/data/englishLevel.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export enum EnglishLevel {
-    B1 = 'B1',
-    B2 = 'B2',
-    C1 = 'C1',
-    C2 = 'C2',
+    B1 = 'B1 (Intermediate)',
+    B2 = 'B2 (Upper-Intermediate)',
+    C1 = 'C1 (Advanced)',
+    C2 = 'C2 (Proficient)',
```
Comment (tatianahutii) [inline frontend/src/app/shared/data/englishLevel.ts:5]: create mapper for this

File: frontend/src/app/shared/data/sex.ts [hunk 3]
```diff
@@ -2,5 +2,5 @@ export enum Sex {
     male = 'Male',
     female = 'Female',
     nonBinary = 'Non Binary',
-    dontWantToSpecify = 'Do not want to specify'
+    dontWantToSpecify = 'Don\'t want to specify'
```
Comment (tatianahutii) [inline frontend/src/app/shared/data/sex.ts:5]: the same here

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/auth/sign-up/sign-up.component.ts (hunk 1 above)
Comment (tatianahutii) [reply frontend/src/app/modules/auth/sign-up/sign-up.component.ts:125]: Let's rework our way to work with enums in a separate card

**Pull request closed without merging**